setup.py

```diff
@@ -1,16 +1,17 @@
+from distutils.version import LooseVersion
 from version import version
 from setuptools import find_packages, setup
-from pkg_resources import parse_version, require, DistributionNotFound
+from pkg_resources import require, DistributionNotFound
 
 
 s3am_version = '2.0'
 
 
 def check_provided(distribution, min_version, max_version=None, optional=False):
     # taken from https://github.com/BD2KGenomics/toil-scripts/blob/master/setup.py
-    min_version = parse_version(min_version)
+    parsed_min_version = LooseVersion(min_version)
```

Comment:
The parsed versions it was printing was really weird... a tuple like `('00000000003', '000000000005', '*final')` instead of `3.5.0`. I felt this was more aesthetically pleasing